src/storage/map.rs

```diff
@@ -68,25 +68,25 @@ where
     }
 }
 
-impl<'a, K, V> Iterator for Iter<'a, K, V> {
+impl<'a, K: Copy, V> Iterator for Iter<'a, K, V> {
```

Comment:
This bound confused me a bit, but I think it's all right since all keys are required to be `Copy`. Else this would blow up somewhere else.